environment/deployments/data-curation/main.tf

```diff
@@ -167,6 +167,70 @@ resource "google_storage_bucket_iam_binding" "dp02-hips-bucket-ro-iam-binding" {
   members = var.hips_service_accounts
 }
 
+// Git LFS Storage Bucket (Prod)
+module "storage_bucket_5" {
+  source        = "../../../modules/bucket"
+  project_id    = module.project_factory.project_id
+  storage_class = "REGIONAL"
+  location      = "us-central1"
+  suffix_name   = ["git-lfs"]
+  prefix_name   = "rubin"
+  versioning = {
+    git-lfs = false
+  }
+  force_destroy = {
+    git-lfs = false
+  }
+  labels = {
+    environment = var.environment
+    application = "giftless"
+  }
+}
+// RO storage access to Git-LFS bucket
+resource "google_storage_bucket_iam_binding" "git-lfs-bucket-ro-iam-binding" {
+  bucket  = module.storage_bucket_5.name
+  role    = "roles/storage.objectViewer"
+  members = var.git_lfs_ro_service_accounts
+}
+// RW storage access to Git-LFS bucket
+resource "google_storage_bucket_iam_binding" "git-lfs-bucket-rw-iam-binding" {
+  bucket  = module.storage_bucket_5.name
+  role    = "roles/storage.objectAdmin"
+  members = var.git_lfs_rw_service_accounts
+}
+
+// Git LFS Storage Bucket (Dev)
+module "storage_bucket_6" {
+  source        = "../../../modules/bucket"
+  project_id    = module.project_factory.project_id
+  storage_class = "REGIONAL"
+  location      = "us-central1"
+  suffix_name   = ["git-lfs-dev"]
+  prefix_name   = "rubin"
+  versioning = {
+    git-lfs-dev = false
+  }
+  force_destroy = {
+    git-lfs-dev = false
+  }
+  labels = {
+    environment = var.environment
+    application = "giftless"
+  }
+}
+// RO storage access to Git-LFS Dev bucket
+resource "google_storage_bucket_iam_binding" "git-lfs-bucket-dev-ro-iam-binding" {
+  bucket  = module.storage_bucket_6.name
+  role    = "roles/storage.objectViewer"
+  members = var.git_lfs_ro_dev_service_accounts
+}
+// RW storage access to Git-LFS Dev bucket
+resource "google_storage_bucket_iam_binding" "git-lfs-bucket-dev-rw-iam-binding" {
+  bucket  = module.storage_bucket_6.name
+  role    = "roles/storage.objectAdmin"
```

Comment:
Oh, also, I don't think you want this to be admin. There's no reason for giftless to take admin actions on the bucket. I think this should only be write permissions.

Reply:
Specifically, I think that's `roles/storage.objectUser`.